src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -187,6 +187,15 @@
                                                     <div class="tab">
                                                         <h3 class="tab-title">Attachment #{{ loop.index }}</h3>
                                                         <div class="tab-content">
+                                                            <p>
+                                                                <a href="data:{{ attachment.contentType|default('application/octet-stream') }};base64,{{ collector.base64Encode(attachment.body) }}" download="{{ attachment.filename|default('attachment') }}">
+                                                                    {% if attachment.filename %}
```

Comment:
```suggestion
                                                                    {% if attachment.filename|default %}
```

Need the `|default` here to prevent error on lower `symfony/mime` versions.